fmf/base.py

```diff
@@ -544,3 +547,58 @@ def node(reference):
                 "No tree node found for '{0}' reference".format(reference))
         # FIXME Should be able to remove .cache if required
         return found_node
+
+    def get_raw_data_node(self):
+        raw_data_node = self
+        id_list = list()
+        while True:
+            raw_data = raw_data_node.raw_data
+            if raw_data:
+                return raw_data_node, id_list
+            elif not raw_data_node.parent:
+                raise utils.RootError("Cannot find root node with raw_data, very strange")
+            else:
+                id_list.insert(0, "/" + raw_data_node.name.rsplit("/")[-1])
+                raw_data_node = raw_data_node.parent
+
+    def node_get_raw_data(self):
```

Comment:
Could this be a private method?

Reply:
This method coud be useful. in case user wants to modify data in theirs way and do not rely on modify.update.
I can document it here. as we've discussed various ways of modifiying current node.
But theoretically it could be also private, because, probably just me and you understand why it is good for. and in case user want another modify mechanism could create new issues and dissuss way how and why.